ix-dev/community/esphome/ix_values.yaml

```diff
@@ -0,0 +1,9 @@
+images:
+  image:
+    repository: ghcr.io/esphome/esphome
```

Comment:
Looks good overall but this image won't support avahi out of the box, right?

Reply:
out of the box, no. Upstream esphome doesnt have it and recommends the environment variable ive used here to enable ping checks. However, ive seen some other builds like [this ](https://github.com/vxav/esphome-avahi) that add avahi to the upstream image. Im not sure if this is that maintainable (this image is already a year old).

Reply:
Not required but would be nice. I found that the ping doesn't work when the dns doesn't resolve. Happy to test the current version when it's available, though.